storage/replica.go

```diff
@@ -1952,7 +1952,7 @@ func (r *Replica) resolveIntents(ctx context.Context, intents []roachpb.Intent,
 			return r.store.DB().Run(b)
 		}
 		if wait || !r.store.Stopper().RunAsyncTask(func() {
-			if err := action; err != nil {
+			if err := action(); err != nil {
```

Comment:
This is the path taken when you're trying to resolve an external intent, but you don't wait for the command to return. We don't actually use it at the moment (things which don't wait only resolve local intents, and for some remote cases where we could in principle avoid waiting still have to for sequence cache cleanup purposes since they want to do that only when successful).
So yes, doh, but didn't actually matter.